tests/Congress.test.ts

```diff
@@ -486,6 +1330,80 @@ describe("Test for the creation a proposal and the voting", () => {
         await client.post(block_url, { block: new_block });
         await block_manager.waitFor(block_manager.getLastBlockHeight(), boa_client);
         assert.strictEqual(JSBI.toNumber(await boa_client.getBlockHeight()), block_manager.getLastBlockHeight());
-        assert.strictEqual(block_manager.getLastBlockHeight(), 10);
+        assert.strictEqual(block_manager.getLastBlockHeight(), 13);
+
+    });
+
+    it("14-15 Create a dummy block", async () => {
+        await createDummyBlock(14);
+        await createDummyBlock(15);
+    });
+
+
+    it("16~19 Create a dummy block", async () => {
+        await createDummyBlock(16);
+        await createDummyBlock(17);
+        await createDummyBlock(18);
+        await createDummyBlock(19);
+    });
+
+    it("Create a block with a height is 20 and the block has 5 enrollments", async () => {
+        // Create key pairs of validators to be added.
+        const add_validators: KeyPair[] = [];
+        iota(key_position, key_position + 5).forEach((n) => {
+            add_validators.push(ValidatorKey.keys(n));
+            key_position++;
+        });
+
+        // One validator requests UTXO to freeze 40,000 boa each.
+        const utxos = await utxo_provider.getUTXO(Amount.multiply(BOA(40_000), add_validators.length));
+
+        // Create a frozen transaction
+        const tx = block_manager.addValidators(add_validators, utxos, gen_keypair);
+        const enrollments: Enrollment[] = [];
+        const bits = block_manager.getBitMask();
+
+        // Create new validator's enrollment data.
+        enrollments.push(...block_manager.getNewEnrollment());
+
+        // Create enrollment data of validators who need re-enrollment among already registered validators.
+        enrollments.push(...block_manager.getReEnrollment());
+
+        // Arrange the enrollment data in ascending order of UTXO.
+        enrollments.sort((a, b) => {
+            return Utils.compareBuffer(a.utxo_key.data, b.utxo_key.data);
+        });
+
+        // Create a new block.
+        const new_block = block_manager.saveBlock([tx], enrollments, bits);
+
+        const block_url = URI(stoa_private_addr).directory("block_externalized").toString();
+        await client.post(block_url, { block: new_block });
+        await block_manager.waitFor(block_manager.getLastBlockHeight(), boa_client);
+        assert.strictEqual(block_manager.getLastBlockHeight(), 20);
+    });
+
+    it("21-22 Create a dummy block", async () => {
+        await createDummyBlock(21);
+        await createDummyBlock(22);
+    });
```

Comment:
```suggestion
    it("16~22 Create a dummy block", async () => {
        await createDummyBlock(16);
        await createDummyBlock(17);
        await createDummyBlock(18);
        await createDummyBlock(19);
        await createDummyBlock(20);
        await createDummyBlock(21);
        await createDummyBlock(22);
    });
```

Reply:
Reenrollment is required at height 20. otherwise we will not be able to get validator for cycle [20-40].

Reply:
Okay